core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/azureModels.scala

```diff
@@ -18,11 +18,18 @@ final case class WsmJobId(value: String) extends AnyVal
 final case class ManagedIdentityName(value: String) extends AnyVal
 final case class BatchAccountName(value: String) extends AnyVal
 
+final case class PostgresName(value: String) extends AnyVal
+final case class LogAnalyticsWorkspaceName(value: String) extends AnyVal
+
 final case class LandingZoneResources(clusterName: AKSClusterName,
```

Comment:
Adding a few more things to LandingZoneResources since I think we'll need them soon/later.